BoxContentSDK/BoxContentSDK/BOXContentSDKErrors.m

```diff
@@ -16,7 +16,7 @@
 NSString *const BOXAuthTokenRequestErrorInvalidRequest = @"invalid_request";
 
 NSString *const BOXAuthErrorUnauthorizedDevice = @"unauthorized_device";
-NSString *const BOXAuthErrorExceededDeviceLimit = @"exceeded_device_limit";
+NSString *const BOXAuthErrorExceededDeviceLimit = @"exceeded_device_limits";
```

Comment:
since we've been checking against exceeded_device_limit (without 's') in Bender, I wonder if it's a typo in Bender or in the API document. Could we confirm with @ryao, @boxtcarpel if they might know the context?